src/main/resources/templates/currentWeather.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <title>Pogoda</title>
+
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet"
+          integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js"
+            integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0"
+            crossorigin="anonymous"></script>
+</head>
+<body>
+<div style="margin: 100px auto; max-width: 1000px; width: 100%">
+    <h1 th:text="|Aktualna pogoda dla miasta: ${cityName}|"></h1>
+    <table class="table">
+        <thead>
+        <tr>
+            <td>Wskaznik</td>
+            <td>Wartość</td>
+        </tr>
+        </thead>
+        <tbody>
+        <tr th:each="indicator: ${indicators}">
+            <td th:text="${indicator.name}">Temp</td>
+            <td th:text="${indicator.value}">26.3</td>
+        </tr>
```

Comment:
Rozwiąż to w taki sposób, żeby były tutaj polskie nazwy. Wielu osobom "humidity" może nic nie mówić